verifier/verifier.go

```diff
@@ -155,7 +289,7 @@ func (v *verifier) Verify(ctx context.Context, desc ocispec.Descriptor, signatur
 		logger.Debug("Skipping signature verification")
 		return outcome, nil
 	}
-	err = v.processSignature(ctx, signature, envelopeMediaType, trustPolicy, pluginConfig, outcome)
+	err = v.processSignature(ctx, signature, envelopeMediaType, trustPolicy.Name, trustPolicy.TrustedIdentities, trustPolicy.TrustStores, pluginConfig, outcome)
```

Comment:
Why not simply pass the `trustPolicy` struct to keep the argument count lower?

Reply:
because trust policy for blob is different from oci and we want to reuse method.